packages/esbuild-plugin-node/src/common.ts

```diff
@@ -32,39 +32,42 @@ export function wrapModule(
   ${originalSource}
 })(...arguments);
 {
-  let mod = module.exports;
-
-  const { satisfies } = require('semver');
-  const { ${oTelInstrumentationClass} } = require('${oTelInstrumentationPackage}');
   const { diag } = require('@opentelemetry/api');
```

Comment:
Not hugely significant but curious, why is this import outside the `try` block but the others are inside?

Reply:
I needed `diag` outside of `try` to use it in `catch`. But I wanted to keep as much as possible inside the `try` to catch also failures in requiring these libraries.